crnd_web_widget_popup_image/static/src/js/image_popup_owl.js

```diff
@@ -0,0 +1,37 @@
+/** @odoo-module **/
+
+import { Component } from "@odoo/owl";
+import { registry } from "@web/core/registry";
+import { standardFieldProps } from"@web/views/fields/standard_field_props";
+
+export class MyNewWidget extends Component {
```

Comment:
Rename widget to `ImagePopup`.

And, possibly, extend standard Odoo's image widget, instead of creating completely new one.